sdcm/utils/k8s.py

```diff
@@ -124,10 +124,20 @@ def dynamic_client(cls, kluster):
     def dynamic_api(cls, kluster, api_version, kind):
         return cls.dynamic_client(kluster).resources.get(api_version=api_version, kind=kind)
 
+    @classmethod
+    def apps_v1_api(cls, kluster):
+        return getattr(kluster, "_k8s_apps_v1_api", None) or k8s.client.AppsV1Api(cls.api_client(kluster))
+
     @classmethod
     def core_v1_api(cls, kluster):
         return getattr(kluster, "_k8s_core_v1_api", None) or k8s.client.CoreV1Api(cls.api_client(kluster))
 
+    @classmethod
+    def list_statefulsets(cls, kluster, namespace=None, **kwargs):
+        if namespace is None:
+            return cls.apps_v1_api(kluster).list_namespaced_stateful_set(watch=False, **kwargs).items
```

Comment:
Other such methods list resources for all namespaces in this case.

Reply:
fixed